docs/dev/new_check_howto.md

```diff
@@ -313,7 +313,7 @@ files:
         type: boolean
         example: false
     # Try uncommenting this template to see what happens!
-    #- template: instances/default
+    #- template: instances/http
```

Comment:
Small nit for realism, but `- template: instances/default` should also be included uncommented since `init_config/default` is included.

Reply:
@yzhan289 To be fair, little of this is realistic. 😉 The idea here is simply to give people an opportunity to _try_ something and see what happens. But no problem, that item can be added.